src/mococrw/padding_mode.h

```diff
@@ -60,6 +60,11 @@ class NoPadding: public RSAPadding {
 public:
     virtual ~NoPadding() = default;
 
+    NoPadding(openssl::DigestTypes hashingFunction = openssl::DigestTypes::SHA256)
```

Comment:
Why? NoPadding should not have a hash function. This is supposed to configure padding modes. NoPadding has no attributes. Looks a bit like misuse to me.